packages/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -96,5 +96,7 @@ export async function loadAsync(
   if (reporter) {
     defaultConfig = { ...defaultConfig, reporter };
   }
+  const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+  const { loadConfig } = require(resolveModule('metro-config', projectRoot, exp));
```

Comment:
Would be useful to get the path with `projectHasModule('metro-config', projectRoot, exp)` and if the returned path is nullish then throw an error informing the user how to install the package.